openapi/swagger/fs/usr/share/nginx/html/helper/index.html

```diff
@@ -99,6 +99,9 @@ <h4>Query/retrieve data</h4>
           <li>
             <a href="/swagger/?urls.primaryName=flowCytometryUser">flowCytometryUser</a> - API endpoints for retrieving only flow cytometry data or metadata
           </li>
+          <li>
+              <a href="/swagger/?urls.primaryName=filesUser">filesUser</a> - API endpoints for retrieving files
```

Comment:
```suggestion
              <a href="/swagger/?urls.primaryName=filesUser">filesUser</a> - API endpoints for retrieving data or metadata of files, attached to a study
```
This is a my suggestion, but let's approve with product guys